accountsservice/io.elementary.pantheon.AccountsService.xml

```diff
@@ -15,6 +15,17 @@
     <property name="TimeFormat" type="s" access="readwrite">
       <annotation name="org.freedesktop.Accounts.DefaultValue.String" value="24h"/>
     </property>
+    
+    <!--
+      PrefersColorScheme:
+
+      What color scheme the user prefers for the UI.
+      Valid preferences: 0 = No Preference, 1 = Prefer Dark, 2 = Prefer Light.
+      Unrecognized preferences should be considered equal to No Preference.
+    -->
+    <property name="PrefersColorScheme" type="u" access="readwrite">
```

Comment:
```suggestion
    <property name="PrefersColorScheme" type="i" access="readwrite">
```
unsigned integers are for flags, here it is an enum and that is represented by a signed integer :)

Reply:
@tintou this PR is just backporting https://github.com/elementary/default-settings/pull/166

I used unisnged in because it said to here: https://dbus.freedesktop.org/doc/dbus-api-design.html#type-system

>For APIs being used in constrained situations, enumerated values should be transmitted as unsigned integers.